README.md

```diff
@@ -14,6 +14,20 @@ To start your application in the dev profile, simply run:
 
 For further instructions on how to develop with JHipster, have a look at [Using JHipster in development][].
 
+### Doing API-First development using openapi-generator
+
+[OpenAPI-Generator]() is configured for this application. You can generate API code from the `src/main/resources/swagger/api.yml` definition file by running:
+
+```bash
+./gradlew openApiGenerate
+```
+
+Then implements the generated delegate classes with `@Service` classes.
+
+To edit the `api.yml` definition file, you can use a tool such as [Swagger-Editor](). Start a local instance of the swagger-editor using docker by running: `docker-compose -f src/main/docker/swagger-editor.yml up -d`. The editor will then be reachable at [http://localhost:7742](http://localhost:7742).
```

Comment:
Please use default port 8713 as in ports mapping: https://github.com/xm-online/xm-online/wiki/Ports-reservation-map

Reply:
7743 is the port for the Swagger editor. Port 8713 is already in use for ms-tmf-document. Do I need to change the Swagger-Editor port to 8713?